Update hello-hybrid-cloud-kerberos-trust-provision.md (#11400)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md

1. For hybrid Azure AD joined devices, sign out and sign in to the device using Windows Hello for Business.

> [!NOTE]

> > For hybrid Azure AD joined devices, users must perform the first sign in with new credentials while having line of sight to a DC. Without line of sight to a DC, even when the client is configured to use cloud Kerberos trust, the system will fall back to key trust if cloud Kerberos trust login fails.

Reviewer: ```suggestion
> For hybrid Azure AD joined devices, users must sign in the first time with new credentials while having a line of sight to a DC. Without line of sight to a DC, even when the client is configured to use cloud Kerberos trust, the system will fall back to key trust if signing in to cloud Kerberos trust fails.
```

> 

> > For Azure AD joined devices, a line of sight to a DC is not required during the first sign in with new credentials, and is only required in order to access on-premises resources. 

Reviewer: ```suggestion
> For Azure AD joined devices, a line of sight to a DC is not required when first signing in with new credentials, and is only required in order to access on-premises resources. 
```

> [!NOTE]

> > For hybrid Azure AD joined devices, users must perform the first sign-in with new credentials while having line of sight to a DC.
> > 
> > For Azure AD joined devices, a line of sight to a DC is not required during the first sign in with new credentials, and is only required in order to access on-premises resources.  

Reviewer: ```suggestion
> For hybrid Azure AD joined devices, users must sign in the first time with new credentials while having line of sight to a DC.
> 
> For Azure AD joined devices, a line of sight to a DC is not required when first signing in with new credentials, and is only required in order to access on-premises resources.  
```
